backend/src/main/java/hu/flowacademy/konyhatunder/controller/CategoryController.java

```diff
@@ -27,5 +28,10 @@
         return categoryService.findById(id);
     }
 
-
+    @CrossOrigin
+    @PostMapping
+    @ResponseStatus(HttpStatus.ACCEPTED)
```

Comment:
Sima OK jobb lenne, az accepted nem azt jelenti, amit gondolnánk, hanem, hogy még nem fejeződött be a művelet vagy még el sem kezdődött:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
"202 Accepted response status code indicates that the request has been accepted for processing, but the processing has not been completed; in fact, processing may not have started yet."

Reply:
Done.

Reply:
Király, a created tök jó ötlet erre!